src/views/NavBarView.js

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
 import {Query} from 'react-apollo';
 import {GET_STUDY_BY_ID} from '../state/queries';
-import {LoadingPlaceholder} from '../components/Loading';
```

Comment:
Did the study header loading state get added in this PR? If so can we move those changes to another PR